templates/web/fixmystreet.com/header_logo.html

```diff
@@ -0,0 +1,19 @@
+<a href="[% c.cobrand.base_url IF admin %]/" id="site-logo" title="FixMyStreet">
+<svg width="175" height="35" viewBox="0 0 175 35" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Presumably this `<svg>` could be `site-logo-default.svg` (which isn't used?) and then included with an `INSERT`? Then it's only in one place.

Reply:
As you spotted, the `web/cobrands/fixmystreet.com/site-logo-*.svg` files aren’t actually linked to from anywhere. I checked them in just to match the `{.png,@2x.png,.svg}` pattern for all the other images, and in case someone, in future, wanted to use the SVGs for something ;-) I guess we could remove them if you feel they’re gonna confuse people?

I'd only want to `INSERT` an external `.svg` file into the HTML page if we could make it clear that it isn’t just a straight SVG, but one that includes the `<switch>`, `<foreignObject>`, and IE conditional comment hacks that [make SVG inlining work](https://codepen.io/tigt/post/inline-svg-fallback-without-javascript-take-2). But I agree, it’d be nice not to just have this huge chunk of SVG in the middle of the HTML template. Maybe I should name it `site-logo-with-fallback.svg` or something.